ibm_catalog.json

```diff
@@ -390,18 +409,16 @@
               ]
```

Comment:
Please recheck the COS input mappings. Now the variable `use_existing_resource_group` is not being used anywhere.